scripts/backend_tests_incomplete_coverage.txt

```diff
@@ -77,6 +77,8 @@ core.jobs.job_test_utils_test
 core.jobs.job_utils_test
 core.domain.image_validation_services_test
 scripts.linters.js_ts_linter_test
+core.storage.improvements.gae_models_test
+core.storage.audit.gae_models_test
 core.storage.story.gae_models_test
```

Comment:
I think you have to remove this file as you have increased the backend coverage to 100% for this file only.

Reply:
Done!